app/models/region.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: regions
+#
+#  id                                         :uuid             not null, primary key
+#  claims_funding_available_per_hour_currency :string           default("GBP"), not null
+#  claims_funding_available_per_hour_pence    :integer          default(0), not null
+#  name                                       :string           not null
```

Comment:
Can we add validation for `:name` presence? Same for `:claims_funding_available_per_hour_currency` and `:claims_funding_available_per_hour_pence`